<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainQssWidget</class>
 <widget class="QWidget" name="MainQssWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTextEdit" name="m_pTEQSS"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>边框</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="m_pBtnType">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>背景</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="m_pBtnBorder">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="m_pBtnState">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="m_pBtnBackground">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="m_pBtnFont">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>字体</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="m_pMainStackedWgt">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="type">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>类型设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>选择类型</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_pCBType"/>
             </item>
             <item>
              <widget class="QLineEdit" name="m_pLECustomType"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="state">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>状态设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>状态选择</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_pCBState"/>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="border">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>边框设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pCBIsShowBorder">
            <property name="text">
             <string>是否显示边框</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWgtBorderConfig" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>tips:设置背景图片时无法同时设置边框</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>边框种类</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="m_bCBBorderType"/>
             </item>
             <item>
              <widget class="QWidget" name="m_pWgtBorderWidth" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>边框宽度</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="m_pSBBorderWidth"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="m_pWgtBorderColor" native="true">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="m_pLEBorderR"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="m_pLEBorderG"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>r</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>g</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="m_pLEBorderB"/>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>边框颜色</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>b</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="background">
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>背景设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>背景颜色</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pCBIsBackgroundColor">
               <property name="text">
                <string>是否设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWgtBackgroundColor" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>r:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="m_pLEBackgroundRed"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>g:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="m_pLEBackgroundGreen"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>b:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="m_pLEBackgroundBlue"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>背景图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pCBIsBackgroundPic">
               <property name="text">
                <string>是否设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWgtBackgroundPic" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLineEdit" name="m_pLEPicPath"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>手动选择</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="preview">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_pLabPreviewError">
            <property name="text">
             <string>其他种类控件不支持预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWgtPreview" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QStackedWidget" name="m_pPreviewStackWgt">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <widget class="QWidget" name="page_2">
                <widget class="QPushButton" name="m_pPreviewBtn">
                 <property name="geometry">
                  <rect>
                   <x>100</x>
                   <y>120</y>
                   <width>80</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>预览</string>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_3">
                <widget class="QLabel" name="label_22">
                 <property name="geometry">
                  <rect>
                   <x>110</x>
                   <y>170</y>
                   <width>54</width>
                   <height>12</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>预览</string>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_4">
                <widget class="QLineEdit" name="lineEdit">
                 <property name="geometry">
                  <rect>
                   <x>120</x>
                   <y>180</y>
                   <width>113</width>
                   <height>20</height>
                  </rect>
                 </property>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="font">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>字体设置</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="m_pCBIsFont">
            <property name="text">
             <string>是否设置字体</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="m_pWgtFont" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="3" column="1">
              <widget class="QLineEdit" name="m_pLEFontB"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>字体g</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="m_pLEFontG"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="m_pLEFontR"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>字体r</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>字体b</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>字号</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="m_pSBFontSize"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="m_pBtnGlobalSettings" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1">
          <widget class="QPushButton" name="m_pBtnImport">
           <property name="text">
            <string>导入</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>导出</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="m_pBtnPreview">
           <property name="text">
            <string>预览</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="m_pBtnClose">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_pCheckBoxDQM">
           <property name="text">
            <string>带双引号</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>全局设置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="m_pBtnUpdate">
           <property name="text">
            <string>手动更新</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>前后缀设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
